index.html

```diff
@@ -1,230 +1,213 @@
-            </p>
-          </section>
-		   <section>
-              <div class="lest-connect-text">
-                <p> LET'S CONNECT </p>
-              </div>
-              <ul class="link-media">
-                <li><a href="https://twitter.com/" target="_blank"><img src="./image/Vector-twitter.svg" alt="twitter icon"></a></li>
-                <li><a href="https://www.linkedin.com/in/jean-walner-alexis-b05539175/" target="_blank"><img src="./image/linkedin-icon.svg" alt="Linkedin icon"></a></li>
-                <li><a href="https://github.com/AlexisJW" target="_blank"><img src="./image/Vector-m.svg" alt="icon"></a></li>
-                <li><a href="https://github.com/AlexisJW" target="_blank"><img src="./image/Vector-github.svg" alt="Github icon"></a></li>
-                <li><a href="https://web.facebook.com/profile.php?id=100001094813889" target="_blank"><img src="./image/Vector.svg" alt="Fingers icon"></a></li>
-              </ul>
-			  <button class="see-project download-resume"> Get my Resume </button>
-          </section>
-          <section>
-            <div class="all-skills-container">
-              <ul>
-                <li class="global-skills-title">
-                  <p> Languages </p>
-                  <div class="down-arrow"></div>
-                </li>
-                <li>
-                  <ul class="global-skills-secondary">
-                    <li>
-                      <img src="./image/icon-javascript.png">
-                      <p> JavaScript </p>
-                    </li>
-                    <li>
-                      <img src="./image/html-icon.png">
-                      <p> HTML </p>
-                    </li>
-                    <li>
-                      <img src="./image/css-icon.png">
-                      <p> CSS </p>
-                    </li>
-                  </ul>
-                </li>
-                <li class="global-skills-title list-closed">
-                  <p> Frameworks </p>
-                  <div class="right-arrow"></div>
-                </li>
-                <li class="global-skills-title list-closed">
-                  <p> Skills </p>
-                  <div class="right-arrow"></div>
-                </li>
-              </ul>
-            </div>
-          </section>
-        </div>
+  
+      <ul class="language-program">
+           <li>
+              <span class="language-title"> LANGUAGES </span>
+              <button class="chevron-button">
+                 <img src="./image/chevron-down.png" alt="" class="chevron">
+              </button>
+			  <ul class="language-list">
+				  <li><img src="./image/JS-icon.png" alt="" class="language-icon"><span class="language-name"> JavaScript </span></li>
+				  <li><img src="./image/HTML5-icon.png" alt="" class="language-icon"><span class="language-name"> HTML </span></li>
+				  <li><img src="./image/CSS-icon.png" alt="" class="language-icon"><span class="language-name"> CSS </span></li>
```

Comment:
- Hey, the image name is `css-icon.png` and not in uppercase. kindly fix this to have your image show up correctly.

Reply:
### SOCIAL MEDIA ICON SPACE

- Please, let's improve our design by adding the correct spaces between the icons to match the design template as much as you can. You can refer to the Figma template to implement this. see your image space below:

![Screenshot from 2022-12-09 16-46-05](https://user-images.githubusercontent.com/59397159/206740287-af59cc11-294b-4f30-ae42-35b5f2d6f66a.png)